docs/content/datasources.md

```diff
@@ -191,6 +192,34 @@ $ echo '{{ (ds "foo" "/second/p1").Value }}' | gomplate -d foo=aws+smp:///foo/
 aaa
 ```
 
+## Using `aws+sm` datasource
+
+### URL Considerations
+For `aws+sm`, only the _scheme_ and _path_ components are necessary to be defined. Other URL components are ignored.
+
+### Output
+
+The output will be the SecretString from the `GetSecretValueOutput` object from the [AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/api/service/secretsmanager/#GetSecretValueOutput)
+
+### Examples
+
+Given your [AWS account's Secret Manager](https://eu-central-1.console.aws.amazon.com/secretsmanager/home?region=eu-central-1#/listSecrets) has the following data:
+
+- `/foo/first/others` - `Bill,Ben` (a StringList)
+- `/foo/first/password` - `super-secret` (a SecureString)
+- `/foo/second/p1` - `aaa`
+
+```console
+$ echo '{{ (ds "foo") }}' | gomplate -d foo=aws+sm:///foo/first/password
```

Comment:
This example section needs to be revisited based on adjustments to the above data